pages/config.py

```diff
@@ -1,6 +1,24 @@
 import PySimpleGUI as sg
 import datetime
 import configparser
+from helpers.utils import getCnpj
+
+def default_year():
+    with open("env.txt", "r") as configFile:
+        content = configFile.readlines()
+        yearInFile = [i for i in content if "year =" in i]
+        return yearInFile[0][7:-1]
+
+
+def default_month():
+    with open("env.txt", "r") as configfile:
```

Comment:
m helpers.utils já existe uma função que lê o arquivo e retorna o mês que está no arquivo.
Então o adequado seria utilizar essa função que já supri a necessidade.

Reply:
Mesma coisa, a função não funciona. Vou tentar novamente alterando ela

Reply:
A função retorna um int mas precisamos de um string com o nome do mês
def getMonth():
    with open("env.txt", "r") as configFile:
        content = configFile.readlines()
        monthInFile = [i for i in content if "month =" in i][0]
        containsData = monthInFile[8:-1]  != ''

        if containsData and int(monthInFile[8:-1]) > 0:
            return int(monthInFile[8:-1])
        else:
            return 0
       
Por isso o uso da nova função
def default_month():
    with open("env.txt", "r") as configfile:
        lines = configfile.readlines()
        monthInFile = [i for i in lines if "month =" in i][0]
        month_num = int(monthInFile.split('=')[1].strip())
        months = ['Janeiro', 'Fevereiro', 'Março', 'Abril', 'Maio', 'Junho', 'Julho', 'Agosto', 'Setembro', 'Outubro',
                  'Novembro', 'Dezembro']
        return months[month_num - 1]
        
Verificar uma forma de a função inicial servir para os dois propósitos?